### PR TITLE
feat: added alerts

### PR DESCRIPTION
**Pull request by arjundevn**
Added the "Alerts" component of the component library

### CODE REVIEW

**Comment by arjundevn** [comment]
alerts

File: components/alerts/alerts.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          <a href="#"><li>Image</li></a>
+          <a href="#"><li>Input</li></a>
+          <a href="#"><li>Text Utilities</li></a>
+          <a href="#"><li>List</li></a>
+          <a href="#"><li>Navigation</li></a>
+          <a href="#"><li>Modal</li></a>
+          <a href="#"><li>Rating</li></a>
+          <a href="#"><li>Toast</li></a>
+          <a href="#"><li>Simplified Grid</li></a>
+        </ul>
```
Comment (sks-21) [inline components/alerts/alerts.html]: a tags should be inside li tag

File: components/alerts/alerts.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
+.alert {
+  border-radius: 5px;
+  height: 2rem; 
+  margin:2rem;
+  padding:1rem;
+  color: black;
```
Comment (sks-21) [inline components/alerts/alerts.css:15]: use generic color variables

File: components/alerts/alerts.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+.primary-alert {
+  background-color: rgb(170, 182, 238);
+}
+
+.success-alert {
+  background-color: rgb(170, 238, 198);
+}
```
Comment (sks-21) [inline components/alerts/alerts.css]: you may use hex color codes

File: index.html [hunk 4]
```diff
@@ -21,7 +21,7 @@
       </div>
       <div class="components-list">
         <ul>
-          <a href="alerts.html"><li>Alert</li></a>
+          <a href="./components/alerts/alerts.html"><li>Alert</li></a>
```
Comment (sks-21) [inline index.html]: same as above

**Review by sks-21 — APPROVED** [review]
Go through the suggestions once. Rest everything looks good.

**Review by shaksham08 — APPROVED** [review]
Nice use of dark theme

**Pull request merged**